### ISSUE TITLE
does the inference process need image and imane_shape?

### ISSUE THREAD

**Issue by zhb0920** [traveller59/second.pytorch#47]

**Comment by traveller59** [comment]
no, but draw 2d bbox need image_shape to convert lidar box to camera coordination.

**Comment by zhb0920** [comment]
> no, but draw 2d bbox need image_shape to convert lidar box to camera coordination.

How can i  inference without  image and imane_shape?
